OpenRA.Mods.yupgi_alert/OpenRA.Mods.yupgi_alert.csproj

```diff
@@ -2,9 +2,7 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>10.0.0</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
```

Comment:
The linked comment mentions that there can be syncing issues between x86 and x64 compilations. I guess that applies to mod dlls though, sorry for the confusion.